yamllint.yml

```
@@ -0,0 +1,6 @@
+---
+extends: default
+rules:
+  line-length: disable
+  braces: { min-spaces-inside: 1, max-spaces-inside: 1 }
+  brackets: { min-spaces-inside: 1, max-spaces-inside: 1 }
```

If there are still some questions on true/false vs. yes/no, then maybe ignoring these issues would be the best for now.

```suggestion
  brackets: { min-spaces-inside: 1, max-spaces-inside: 1 }
  truthy: disable
```